app/uk/gov/hmrc/gform/sharedmodel/formtemplate/SummarySection.scala

```diff
@@ -26,7 +26,8 @@ case class SummarySection(
   header: SmartString,
   footer: SmartString,
   continueLabel: Option[SmartString],
-  fields: Option[NonEmptyList[FormComponent]]
+  fields: Option[NonEmptyList[FormComponent]],
+  default: Option[Boolean] = None
```

Comment:
Why is this field called `default` and why it is optional `Boolean`?

Reply:
If the summary section is not present,  the backend generates a default summary section. It needs to know which is default or not. Right, it can be represented by only Boolean without Option.

Reply:
but if we do it as Boolean, we need to put it in JSON.

Reply:
but if we do it as a boolean, the `default` attribute will be mandatory in JSON. I think it should be optional to ignore in Json